pachyderm/templates/etcd/statefulset.yaml

```diff
@@ -1,4 +1,6 @@
----
+{{- if le .Values.etcd.dynamicNodes 0.0 -}}
+{{ fail "dynamicNodes must be > 0" }}
```

Comment:
If we're failing here for no nodes, we should probably remove the dymanic nodes check around the service above